packages/core/core/src/public/Bundle.js

```diff
@@ -73,14 +75,28 @@ export class Bundle implements IBundle {
     return this.#bundle.assetGraph.getTotalSize(asset);
   }
 
-  traverseAssets<TContext>(
-    visit: GraphTraversalCallback<Asset, TContext>
-  ): ?TContext {
-    return this.#bundle.assetGraph.traverse((node, ...args) => {
-      if (node.type === 'asset') {
-        return visit(node.value, ...args);
+  resolveSymbol(asset: Asset, symbol: Symbol) {
+    return this.#bundle.assetGraph.resolveSymbol(asset, symbol);
+  }
+
+  traverseAssets<TContext>(visit: GraphVisitor<Asset, TContext>): ?TContext {
+    return this.#bundle.assetGraph.traverseAssets(visit);
+  }
+
+  traverseAncestors(startNode: TNode, visit: GraphVisitor<TNode, TContext>) {
+    return this.#bundle.assetGraph.traverseAncestors(startNode, visit);
+  }
+
+  hasChildBundles() {
```

Comment:
Probably should be in the BundleGraph instead, but again the packager doesn't have access to the BundleGraph...

Reply:
now it does :)